package.json

```diff
@@ -0,0 +1,99 @@
+{
+  "name": "kubevirt-web-ui",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@storybook/addon-actions": "^3.4.10",
+    "@storybook/addon-options": "^3.4.10",
+    "@storybook/react": "^3.4.10",
+    "autoprefixer": "7.1.6",
+    "babel-core": "6.26.0",
+    "babel-eslint": "7.2.3",
+    "babel-jest": "20.0.3",
+    "babel-loader": "7.1.2",
+    "babel-preset-react-app": "^3.1.1",
+    "babel-runtime": "6.26.0",
+    "case-sensitive-paths-webpack-plugin": "2.1.1",
+    "chalk": "1.1.3",
+    "css-loader": "0.28.7",
+    "dotenv": "4.0.0",
+    "dotenv-expand": "4.2.0",
+    "eslint": "4.10.0",
+    "eslint-config-react-app": "^2.1.0",
+    "eslint-loader": "1.9.0",
+    "eslint-plugin-flowtype": "2.39.1",
+    "eslint-plugin-import": "2.8.0",
+    "eslint-plugin-jsx-a11y": "5.1.1",
+    "eslint-plugin-react": "7.4.0",
+    "extract-text-webpack-plugin": "3.0.2",
+    "file-loader": "1.1.5",
+    "fs-extra": "3.0.1",
+    "html-webpack-plugin": "2.29.0",
+    "jest": "20.0.4",
+    "object-assign": "4.1.1",
+    "patternfly-react": "^2.14.0",
+    "postcss-flexbugs-fixes": "3.2.0",
+    "postcss-loader": "2.0.8",
+    "promise": "8.0.1",
+    "raf": "3.4.0",
+    "react": "^16.4.2",
+    "react-dev-utils": "^5.0.1",
+    "react-dom": "^16.4.2",
+    "resolve": "1.6.0",
+    "style-loader": "0.19.0",
+    "sw-precache-webpack-plugin": "0.11.4",
+    "url-loader": "0.6.2",
+    "webpack": "3.8.1",
```

Comment:
imho many of these needs to be dev-dep, not to mention there are really a lot here that I'm not sure we need, plus its an old version of webpack, which I'm not sure we should start with?

Reply:
Full agreement with the dev-deps.

I think this is what was generated by most recent react-create-app, right?
Considering project context, sounds reasonable to me to stay aligned with openshift/console on that.
IIRC, patternfly-react has no requirement on particular webpack version.

Reply:
Lesson learned from other projects: we should keep updating packages and corresponding sources as often as possible. In small steps. Otherwise it becomes nightmare very quickly. 
We should agree on process for that - once a week? By human is enough, IMO.

Reply:
once a week is a bit too often for a human I'd say. I'd suggest to do it at the beginning of each sprint (e.g. once in 3 weeks). This way we don't risk to break it just before the iteration ends :)

Reply:
Right, makes sense

Reply:
there are tools to automate it(e.g. https://greenkeeper.io/), while not sure i love  that option, i think a good way is to keep version matrix relaxed on main branch, and lock it for stable releases, this way you fix issues quickly (assuming its easy fix) along the way, but keep stable branches stable over time.

Reply:
Guys, all of this is currently taken from create-react-app template :smile: 

> imho many of these needs to be dev-dep

Agreed, it makes sense to separate developer/tool deps from build deps.

> not to mention there are really a lot here that I'm not sure we need

Yes, we will simplify/remove & align deps with other projects. With this PR, @rawagner can start experimenting with "Create/Import VM" wizard component in Storybook, while we can update/simplify/improve the project structure.

> plus its an old version of webpack, which I'm not sure we should start with?

webpack 4.x released on Feb 2018 but there might still be plugins/loaders which don't work 100% with the new version. For a brand new project, I agree that it makes sense to use the latest stable version, so we'll upgrade webpack version as a follow-up. (Fun fact, [ovirt-web-ui uses webpack 1.x](https://github.com/oVirt/ovirt-web-ui/blob/master/package.json#L63) and it might not yield very optimized JS output, but it still works.)

> I think this is what was generated by most recent react-create-app, right?

Exactly! :grin: 

> Lesson learned from other projects: we should keep updating packages and corresponding sources as often as possible. In small steps. Otherwise it becomes nightmare very quickly.
> I'd suggest to do it at the beginning of each sprint (e.g. once in 3 weeks). This way we don't risk to break it just before the iteration ends :)

Agreed. As @ohadlevy mentioned, there are tools like GitHub [Greenkeeper](https://github.com/integration/greenkeeper) to monitor & update-test your project's dependencies, we can use that.